### PR TITLE
Block account

### LINKED ISSUE #125: Possibility to block an account (different that locking)
We would like to manage two different status for an account:
* Lock: the password is locked, but a user can still reset its password (for OpenLDAP, it is linked to pwdAccountLockedTime)
* Block: the account is no more useable, even if the password is still valid (for OpenLDAP, it is linked to pwdEndTime)

### PR DESCRIPTION
**Pull request by davidcoutadeur**
Closes #125

### CODE REVIEW

**Comment by davidcoutadeur** [comment]
To discuss: add also the possibility the add a comment while enabling / disabling account

File: htdocs/disableaccount.php [hunk 1]
```diff
@@ -38,7 +44,7 @@
 }
 
 if ($audit_log_file) {
-    auditlog($audit_log_file, $dn, $audit_admin, "disableaccount", $result);
+    auditlog($audit_log_file, $dn, $audit_admin, "disableaccount", $result, $comment);
```
Comment (coudot) [inline htdocs/disableaccount.php:47]: Comment modal is only defined for now in lock/unlock panel in the HTML template.

We may add comment for all action, but this should maybe done in a separate PR.

File: htdocs/display.php
Comment (coudot) [inline htdocs/display.php:1]: Seems some cleaning has to be done, as I forgot to remove OpenLDAP specific code. With the merge of AD support branch, we should have no more any reference to an OpenLDAP specific attribute.

Maybe this should also be part of another PR.

File: htdocs/enableaccount.php [hunk 2]
```diff
@@ -38,7 +44,7 @@
 }
 
 if ($audit_log_file) {
-    auditlog($audit_log_file, $dn, $audit_admin, "enableaccount", $result);
+    auditlog($audit_log_file, $dn, $audit_admin, "enableaccount", $result, $comment);
```
Comment (coudot) [inline htdocs/enableaccount.php:47]: Same than previous comment, this need an adaptation of HTML template,

File: htdocs/disableaccount.php (hunk 1 above)
Comment (davidcoutadeur) [reply htdocs/disableaccount.php:47]: Maybe I'll do another PR for the comment feature...

Anyway, the last argument of the function auditlog is needed, else Service-Desk crashes.

File: htdocs/display.php
Comment (coudot) [reply htdocs/display.php:1]: This was done in #170

**Comment by davidcoutadeur** [comment]
This feature seems ready now. merging
